stl/inc/functional

```diff
@@ -795,7 +795,11 @@ private:
 constexpr size_t _Space_size = (_Small_object_num_ptrs - 1) * sizeof(void*);
 
 template <class _Impl> // determine whether _Impl must be dynamically allocated
-_INLINE_VAR constexpr bool _Is_large = (_Space_size < sizeof(_Impl)) || !_Impl::_Nothrow_move::value;
+_INLINE_VAR constexpr bool _Is_large = (_Space_size < sizeof(_Impl)) || !_Impl::_Nothrow_move::value
+#ifdef __cpp_aligned_new

```

Comment:
> in C++14 mode could feasibly override `operator new` to provide `X` byte alignment

I think it would not work. `std::function` is written to call global `operator new`.

Reply:
Should I make function using class `operator new`? This is more significant change then.

Reply:
> could feasibly override `operator new`

I should have said "could feasibly _replace_ `operator new`" here.

> Should I make function using class `operator new`?

Please no. =)

Reply:
So, do you mean user replaces _global_ `operator new`, just to over-align a small function object? Well, it is weird, but legitimate.

Reply:
Yes: I'm not claiming this is a common or recommended use case, I just think we should do the thing that has a sliver of a possibility of being correct instead of the one we know is wrong.

Reply:
Addressed by disabling `SFO` for over-aligned unconditionally. (and not doing anything else regarding `operator new`)